common/app/views/support/Commercial.scala

```diff
@@ -224,8 +224,11 @@ object Commercial {
       ) mkString " | "
     }
 
-    def mkInteractionTrackingCode(containerIndex: Int, cardIndex: Int, card: ContentCard): String = Seq(
+    def mkInteractionTrackingCode(containerIndex: Int, cardIndex: Int, card: ContentCard, containerDisplayName: Option[String])(implicit request: RequestHeader): String = Seq(
+      containerDisplayName.getOrElse("unknown container"),
+      Edition(request).id,
```

Comment:
I think `Edition` might be the top-level "thing", eg. `edition | container-index | card-index | headline` seem to go together, followed by container name and branding . Any kind of sensible grouping will be fine, I think. Or is ordering not important to us?